provider/datagen/Cargo.toml

```diff
@@ -78,6 +78,7 @@ toml = "0.5"
 writeable = { version = "0.5.1", path = "../../utils/writeable" }
 zerovec = { version = "0.9.4", path = "../../utils/zerovec", features = ["serde", "yoke"] }
 zip = { version = ">=0.5, <0.7", default-features = false, features = ["deflate"] }
+memchr = "2.5.0"
```

Comment:
Is it worth the new dep?

This should fail one of @Manishearth's deps tests, right?

Reply:
It's already a dep of regex-automata